cleanlab/object_detection/summary.py

```diff
@@ -241,7 +241,7 @@ def get_sorted_bbox_count_idxs(labels, predictions):
 
 
```

Comment:
code is not passing black formatter: https://github.com/cleanlab/cleanlab/actions/runs/8563071951/job/23467504931?pr=1084

please make sure you run `black` at the end before git commit (or install the pre-commit hooks as described in the DEVELOPMENT.md)

Reply:
Good catch, just reformatted using `black`